packages/eslint-plugin/tests/rules/no-empty-interface.test.ts

```diff
@@ -58,6 +58,26 @@ interface Bar extends Foo {}
     },
     {
       code: `
+interface Foo {
+  props: string;
+}
+
+interface Bar extends Foo {}
+
+class Bar {}
```

Comment:
Hmm, only having one test for this new behavior worries me a bit. Let's add some more tests in:

* What happens if there's a class by a different name?
* What if there's a class expression (`const Bar = class Bar {}`)